includes/public/class-wppr-public.php

```diff
@@ -540,20 +542,41 @@ public function show_comment_ratings( $text ) {
 	 * AMP support for WPPR
 	 */
 	public function wppr_amp_support() {
-		$amp_cache_key = '_wppr_amp_css';
-		$cached_css    = get_transient( $amp_cache_key );
-		if ( ! empty( $cached_css ) ) {
-			echo $cached_css;
-
-			return;
-		}
+		$current_template = $this->review->get_template();
 		require_once( ABSPATH . 'wp-admin/includes/file.php' );
 		WP_Filesystem();
 		global $wp_filesystem;
 		$output = '';
 		$output .= $wp_filesystem->get_contents( WPPR_PATH . '/assets/css/frontpage.css' );
-		$output .= $wp_filesystem->get_contents( WPPR_PATH . '/assets/css/circle.css' );
-		$output .= $wp_filesystem->get_contents( WPPR_PATH . '/assets/css/rating-amp.css' );
+		if ( $current_template == 'style1' ) {
```

Comment:
why we need to use 3 transients, we can dynamically build the cache key based on the current_template, something like `_wppr_amp_css_<current_template>` and save the output in that transient, not every file output to a different one.

Reply:
I didn't think of that. Thanks. will make the changes.

Reply:
Made the changes.